site-src/api-types/httproute.md

```diff
@@ -165,6 +165,31 @@ Service:
 Reference the [backendRef][backendRef] API documentation for additional details
 on `weight` and other fields.
 
+#### Timeouts (optional)
+
+??? example "Experimental Channel in v1.0.0+"
+
+    HTTPRoute timeouts are part of the Experimental Channel in `v1.0.0+`.
+
+HTTPRoute Rules now include a `Timeouts` field. If unspecified, timeout behavior is implementation-specific.
+
+There are 2 kinds of timeouts that can be configured in an HTTPRoute Rule:
+
+1. `request` is the timeout for the Gateway API implementation to send a response to a client HTTP request. Whether the Gateway starts the timeout before or after the entire client request stream has been received, is implementation dependent. This timeout is intended to cover as close to the whole request-response transaction as possible, although an implementation MAY choose to start the timeout after the entire request stream has been received instead of immediately after the transaction is initiated by the client.
+
+2. `backendRequest` is a timeout for a single request from the Gateway to a backend. This timeout covers the time from when the request first starts being sent from the gateway to when the full response has been received from the backend. This can be particularly helpful if the Gateway retries connections to a backend.
+
+Because the `request` timeout encompasses the `backendRequest` timeout, the value of `backendRequest` must not be greater than the value of `request` timeout.
+
+Timeouts are optional and their fields are of type `Duration`. A zero-valued timeout ("0s") MUST be interpreted as disabling the timeout. A valid non-zero-valued timeout MUST be >= 1ms.
```

Comment:
Shouldn't this link to the Duration spec?

```suggestion
Timeouts are optional and their fields are of type [Duration](/geps/gep-2257/). A zero-valued timeout ("0s") MUST be interpreted as disabling the timeout. A valid non-zero-valued timeout MUST be >= 1ms.
```

Reply:
Thank @frankbu, maybe this should be referencing the API Specification `#gateway.networking.k8s.io/v1.Duration` instead of GEP-2257. What do you think?

Reply:
Good question. The v1.Duration doc says: `The format is as specified in GEP-2257`, but nowhere is there a link to the actual spec. Should we have a link to GEP-2257 here, there, both, or neither? @robscott WDYT?

Reply:
I don't have a strong opinion here, but tend to lean towards linking directly to the GEP since that's where they'd end up in either case.